src/main/java/ch/njol/skript/doc/HTMLGenerator.java

```diff
@@ -603,6 +609,17 @@ private String generateEvent(String descTemp, SkriptEventInfo<?> info, @Nullable
 		String[] keywords = info.getKeywords();
 		desc = desc.replace("${element.keywords}", keywords == null ? "" : Joiner.on(", ").join(keywords));
 
+		// Cancellable
+		boolean cancellable = false;
+		for (Class<? extends Event> event : info.events) {
```

Comment:
I wonder if it might be good to have a common isCancellable method between this and EffCancelEvent

Reply:
Good idea!